src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -135,7 +134,71 @@ public void exec(final URI home) throws IOException {
     }
 
     /**
+     * BkBasic produces headers with addresses without slashes.
+     *
+     * @throws IOException If some problem inside
+     */
```

Comment:
@alexey-krylov let's add a new puzzle to refactoring this test to use real sockets instead mocked object
see https://github.com/yegor256/takes/pull/426